core/tests/protractor_utils/workflow.js

```diff
@@ -136,6 +130,30 @@ var getExplorationPlaytesters = function() {
   return _getExplorationRoles('viewer');
 };
 
+var expectNoFeedbacks = function() {
```

Comment:
Perhaps we could have something more along the lines of getNumberOfFeedbackMessages()? Then we would get the result of this and compare it with 0.